### ISSUE TITLE
❓Toggle, DropDown & Slider property pane controls missing aria

### ISSUE THREAD

**Issue by andrewconnell** [SharePoint/sp-dev-docs#710]
#### Category
- [x] Question
- [ ] Typo
- [ ] Bug
- [ ] Additional article idea

#### Expected or Desired Behavior
All property pane controls based on underlying Fabric React controls should expose aria properties for accessibility when the underlying Fabric React control does.

Shouldn't the property pane controls expose aria properties for screen readers & accessibility purposes?

#### Observed Behavior
These property pane controls do not have aria properties exposed on their interfaces:

- Dropdown (*[underlying Fabric React control](https://dev.office.com/fabric#Implementation)*)
- Slider (*[underlying Fabric React control](https://dev.office.com/fabric#Implementation)*)
- Toggle (*[underlying Fabric React control](https://dev.office.com/fabric#Implementation)*)

**Comment by chakkaradeep** [comment]
Different question: Are you saying these controls are not accessible and you are unable to use screen readers for accessibility purposes? Can you give us a repro?

Thanks!

**Comment by andrewconnell** [comment]
No… I’m saying looking at all the property pane controls, I noticed a few didn’t have aria properties. Knowing they are based on the Fabric React components, when the underlying Fabric component had an aria property but the SPFx property pane didn’t (the three here), I got curious as to why so posted the question.

I don’t have a repo… just asking how you would set the aria property on the underlying control.

**Comment by pgonzal** [comment]
Accessibility is a priority for SharePoint Framework.  We'll look into implementing this.

**Comment by srideshpande** [comment]
Hello, @andrewconnell - Thanks for raising the issue. This issue is now fixed and will be available in the next release. I am closing the issue now but please feel free to reopen it if you don't see the aria properties on any of the controls in the next release.

Thanks,
Srikanth
